From: Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
Date: Wed, 13 Apr 2016 00:07:56 +0200
Subject: [PATCH] lp#1569589: a more honest attempt at disposing
 SvtFileView_Impl

Dunno yet if that completely fixes the story -- the reference here are
quite messy: ownership isnt exactly obvious.

Change-Id: I54d71ac724590e85549bd152c963fdc8de46746e

---
 svtools/source/contnr/fileview.cxx | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/svtools/source/contnr/fileview.cxx b/svtools/source/contnr/fileview.cxx
--- a/svtools/source/contnr/fileview.cxx
+++ b/svtools/source/contnr/fileview.cxx
@@ -1486,6 +1486,9 @@ SvtFileView_Impl::~SvtFileView_Impl()
 {
     Clear();
     mpView.disposeAndClear();
+    mpCurView.clear();
+    mpIconView.disposeAndClear();
+    mpAntiImpl.clear();
 }
 
 
-- 
2.5.0

